web/components/swipe/swipe-card.tsx

```diff
@@ -0,0 +1,376 @@
+            <SiteLink
+              className="absolute -top-9 z-10"
+              href={contractPath(contract)}
+              followsLinkClass
+            >
+              <div
+                className={clsx(
+                  'mx-4 text-white drop-shadow',
+                  getQuestionSize(question)
+                )}
+              >
+                {question}
+              </div>
+            </SiteLink>
+            <div className="absolute top-28 left-[calc(50%-80px)] z-10 mx-auto">
+              <Percent
+                currPercent={currPercent}
+                yesPercent={yesPercent}
+                noPercent={noPercent}
+                outcome={
+                  buttonAction === 'YES'
+                    ? 'YES'
+                    : buttonAction === 'NO'
+                    ? 'NO'
+                    : action === 'left'
+                    ? 'NO'
+                    : action === 'right'
+                    ? 'YES'
+                    : undefined
+                }
+              />
+            </div>
+            <div className="absolute right-4 bottom-32 z-10">
+              <Actions user={user} contract={contract} />
+            </div>
+
+            <Col className="absolute -bottom-16 z-10 w-full gap-6">
+              <DescriptionAndModal
+                description={description}
+                isModalOpen={isModalOpen}
+                setIsModalOpen={setIsModalOpen}
+              />
+              {swipeBetPanel}
+            </Col>
+            <div className="absolute bottom-0 z-0 min-h-[50%] w-full bg-gradient-to-b from-transparent to-black pb-4" />
+            <img src={image} alt="" className="h-full object-cover" />
+          </div>
+          <div className="h-20 w-full rounded-b-2xl bg-black" />
+        </Col>
+      </>
+    )
+  }
+)
+
+const CornerDetails = (props: { contract: Contract; className?: string }) => {
+  const { contract, className } = props
+  const { creatorName, creatorAvatarUrl, closeTime } = contract
+
+  return (
+    <div className={clsx('m-4 flex gap-2 self-start', className)}>
+      <Avatar size="sm" avatarUrl={creatorAvatarUrl} noLink />
+      <div className="text-xs">
+        <div className="text-white">{creatorName} </div>
+        {closeTime != undefined && (
+          <div className="text-gray-400 ">
+            trading closes {fromNow(closeTime)}
+          </div>
+        )}
+      </div>
+    </div>
+  )
+}
+
+function Actions(props: { user?: User; contract: BinaryContract }) {
+  const { user, contract } = props
+
+  return (
+    <Col className="flex flex-col items-center justify-center">
+      <LikeButton
+        contentId={contract.id}
+        contentCreatorId={contract.creatorId}
+        user={user}
+        contentType={'contract'}
+        totalLikes={contract.likedByUserCount ?? 0}
+        contract={contract}
+        contentText={contract.question}
+        size={'lg'}
```

Comment:
Make bigger?

Reply:
I think it'll look reasonably sized once we get other functionalities alongside it